docs/learn/smart-contract-developers/attach-token-metadata.md

```diff
@@ -0,0 +1,472 @@
+---
+sidebar_label: '💽 Attach Token Metadata'
+sidebar_position: 3
+description: Learn how to set and update LSP4 Metadata of digital assets on LUKSO.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Attach Token Metadata
+
+:::tip Contract Storage
+
+To bring LSP contracts to life, you can use the 🗂️ [ERC725Y storage](../../standards/lsp-background/erc725.md#erc725y-generic-data-keyvalue-store) to set or modify certain data keys and values. Typically, these data keys reflect **any content** apart from the constructor parameters that have been initially set. All standardized storage keys follow the [LSP2 JSON Schema](../../standards/generic-standards/lsp2-json-schema.md).
+
+:::
+
+There are two different variants of modifying ERC725Y storage:
```

Comment:
```suggestion
two different variants of modifying ERC725Y storage"
```

this sentence is more complex than it should be.

"You have two options to edit metadata:"

(but even the option 2 is imo out of the scope of this guide